src/components/Tooltip/index.tsx

```diff
@@ -1,24 +1,57 @@
-import React, { ReactNode } from "react"
+import React, { ReactNode, useEffect } from "react"
 import {
   Popover,
   PopoverArrow,
   PopoverBody,
   PopoverContent,
   PopoverProps,
   PopoverTrigger,
+  useDisclosure,
 } from "@chakra-ui/react"
 
+import { isMobile } from "@/lib/utils/isMobile"
+
 export interface IProps extends PopoverProps {
   content: ReactNode
   children?: ReactNode
 }
 
 const Tooltip: React.FC<IProps> = ({ content, children, ...rest }) => {
+  const { isOpen, onOpen, onClose } = useDisclosure()
+
+  // Close the popover when the user scrolls.
+  // This is useful for mobile devices where the popover is open by clicking the
+  // trigger, not hovering.
+  useEffect(() => {
+    let originalPosition = 0
+
+    const handleScroll = () => {
+      const delta = window.scrollY - originalPosition
+
+      // Close the popover if the user scrolls more than 80px
+      if (isOpen && Math.abs(delta) > 80) {
+        onClose()
+      }
```

Comment:
This works great! I like that it doesn't close from the slightest nudge... it feels natural